Deployment/terraform_conditional_run.ps1

```diff
@@ -52,9 +52,9 @@ Write-output "Terraform output as json"
 $terraformOutput = terraform output -json | ConvertFrom-Json
 
 write-output "Set JSON output into pipeline variables"
-Write-Host "##vso[task.setvariable variable=MsiWebUrl;isOutput=true]$env:SERVICE_DNS_URL"
+Write-Host "##vso[task.setvariable variable=MsiWafUrl;isOutput=true]$env:SERVICE_DNS_URL"
+Write-Host "##vso[task.setvariable variable=AdminMsiWafUrl;isOutput=true]$env:SERVICE_DNS_URL/RadioNavigationalWarningsAdmin"
```

Comment:
Suggest swapping order of "msi" & "admin" as per our discussion earlier for the new URL of the admin site.
```suggestion
Write-Host "##vso[task.setvariable variable=MsiAdminWafUrl;isOutput=true]$env:SERVICE_DNS_URL/RadioNavigationalWarningsAdmin"
```